tests/Microsoft.DotNet.Docker.Tests/SdkImageTests.cs

```diff
@@ -178,41 +177,15 @@ public async Task VerifyDotnetFolderContents(ProductImageData imageData)
             IEnumerable<SdkContentFileInfo> actualDotnetFiles = GetActualSdkContents(imageData);
             IEnumerable<SdkContentFileInfo> expectedDotnetFiles = await GetExpectedSdkContentsAsync(imageData);
 
-            bool hasCountDifference = expectedDotnetFiles.Count() != actualDotnetFiles.Count();
+            using TempFileContext actualFiles = FileHelper.UseTempFile();
+            using TempFileContext expectedFiles = FileHelper.UseTempFile();
```

Comment:
Renaming these variables would provide better clarity.

```suggestion
            using TempFileContext actualFilesContext = FileHelper.UseTempFile();
            using TempFileContext expectedFilesContext = FileHelper.UseTempFile();
```